rootfs/scheduler/__init__.py

```diff
@@ -194,14 +194,22 @@
                 "value":"$appversion"
             },
             {
-                "name": "DOCKERIMAGE",
-                "value":"1"
+                "name": "BUILDER_STORAGE",
+                "value":"$storagetype"
+            },
+            {
+                "name": "DEIS_MINIO_SERVICE_HOST",
```

Comment:
Fair enough, not super thrilled having more information in there but it's required so YOLO!